docs/application/native/guides/device/resource-monitor.md

```diff
@@ -0,0 +1,302 @@
+ | `RESOURCE_MONITOR_SYSTEM_ATTR_CPU_SYS_UTIL`        | double        | Resource attribute for kernel cpu utilization           |
+ | `RESOURCE_MONITOR_SYSTEM_ATTR_PER_CPU_UTIL`        | array(double) | Resource attribute for per-cpu utilization          |
+ | `RESOURCE_MONITOR_SYSTEM_ATTR_PER_CPU_USER_UTIL`   | array(double) | Resource attribute for per-cpu user utilization         |
+ | `RESOURCE_MONITOR_SYSTEM_ATTR_PER_CPU_SYS_UTIL`    | array(double) | Resource attribute for per-cpu kernel utilization       |
+ | `RESOURCE_MONITOR_SYSTEM_ATTR_POSSIBLE_CPU`        | int           | Resource attribute for the number of possible cpus      |
+ | `RESOURCE_MONITOR_SYSTEM_ATTR_ONLINE_CPU`          | int           | Resource attribute for the number of online cpus        |
+
+**Table: Resource Attribute Keys for Disk**
+
+ | Attribute Keys                                     | Data Type | Description                                             |
+ |----------------------------------------------------|-----------|---------------------------------------------------------|
+ | `RESOURCE_MONITOR_DISK_ATTR_NAME`                  | string    | Resource attribute for disk name                        |
+ | `RESOURCE_MONITOR_DISK_ATTR_READ_PER_SEC`          | double    | Resource attribute for read throughput per sec          |
+ | `RESOURCE_MONITOR_DISK_ATTR_WRITE_PER_SEC`         | double    | Resource attribute for write throughput per sec         |
+ | `RESOURCE_MONITOR_DISK_ATTR_READ_TOTAL`            | uint64    | Resource attribute for total read amount                |
+ | `RESOURCE_MONITOR_DISK_ATTR_WRITE_TOTAL`           | uint64    | Resource attribute for total write amount               |
+
+
+ <a name="resource_controls"></a>
+
+The following table lists the required controls for some resources, which are part of the `resource_monitor_ctrl_id_e` enumeration:
+
+**Table: Resource Controls**
+
+ | Type                | Control Keys                                       | Description                                             |
+ |---------------------|----------------------------------------------------|---------------------------------------------------------|
+ | CPU                 | `RESOURCE_MONITOR_CPU_CTRL_CLUSTER_ID`             | Resource controls for setting cpu cluster id            |
+ | BUS                 | `RESOURCE_MONITOR_BUS_CTRL_DEVICE_ID`              | Resource controls for setting bus device id             |
+ | GPU                 | `RESOURCE_MONITOR_GPU_CTRL_DEVICE_ID`              | Resource controls for setting gpu device id             |
+ | Display             | `RESOURCE_MONITOR_DISPLAY_CTRL_DEVICE_ID`          | Resource controls for setting display device id         |
+ | Disk                | `RESOURCE_MONITOR_DISK_CTRL_DEVICE_ID`             | Resource controls for setting disk device id            |
+
+## Related Information
+- Dependencies
+  - Tizen 7.0 and Later
```

Comment:
I think the profiles should also be mentioned.
Ref: https://github.com/Samsung/tizen-docs/blob/master/reviewguide/review_points_guide.md#dependencies
```suggestion
  - Tizen 7.0 and Higher
```